@expo/fingerprint fingerprint:generate with path broken on 0.13.0

### Minimal reproducible example

Using `0.12.4` and passing a path to fingerprint:generate:

```
➜ yarn dlx @expo/fingerprint@0.12.4 fingerprint:generate apps/mobile
➤ YN0000: · Done in 0s 409ms
# Diff here...
```

Using `0.13.0` and passing a path to fingerprint:generate:

```
➜ yarn dlx @expo/fingerprint@0.13.0 fingerprint:generate apps/mobile
➤ YN0000: · Done in 0s 583ms

node:internal/modules/cjs/loader:1228
  throw err;
  ^

Error: Cannot find module 'glob'
Require stack:
- /private/var/folders/00/m6b9k2y13rvgz5dn7y7_dtx00000gn/T/xfs-d992fa39/dlx-75559/node_modules/@expo/fingerprint/build/ProjectWorkflow.js
- /private/var/folders/00/m6b9k2y13rvgz5dn7y7_dtx00000gn/T/xfs-d992fa39/dlx-75559/node_modules/@expo/fingerprint/build/Options.js
- /private/var/folders/00/m6b9k2y13rvgz5dn7y7_dtx00000gn/T/xfs-d992fa39/dlx-75559/node_modules/@expo/fingerprint/build/Fingerprint.js
- /private/var/folders/00/m6b9k2y13rvgz5dn7y7_dtx00000gn/T/xfs-d992fa39/dlx-75559/node_modules/@expo/fingerprint/build/index.js
- /private/var/folders/00/m6b9k2y13rvgz5dn7y7_dtx00000gn/T/xfs-d992fa39/dlx-75559/node_modules/@expo/fingerprint/cli/build/runLegacyCLIAsync.js
- /private/var/folders/00/m6b9k2y13rvgz5dn7y7_dtx00000gn/T/xfs-d992fa39/dlx-75559/node_modules/@expo/fingerprint/cli/build/cli.js
- /private/var/folders/00/m6b9k2y13rvgz5dn7y7_dtx00000gn/T/xfs-d992fa39/dlx-75559/node_modules/@expo/fingerprint/bin/cli.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Function._load (node:internal/modules/cjs/loader:1055:27)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (/private/var/folders/00/m6b9k2y13rvgz5dn7y7_dtx00000gn/T/xfs-d992fa39/dlx-75559/node_modules/@expo/fingerprint/build/ProjectWorkflow.js:10:16)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/private/var/folders/00/m6b9k2y13rvgz5dn7y7_dtx00000gn/T/xfs-d992fa39/dlx-75559/node_modules/@expo/fingerprint/build/ProjectWorkflow.js',
    '/private/var/folders/00/m6b9k2y13rvgz5dn7y7_dtx00000gn/T/xfs-d992fa39/dlx-75559/node_modules/@expo/fingerprint/build/Options.js',
    '/private/var/folders/00/m6b9k2y13rvgz5dn7y7_dtx00000gn/T/xfs-d992fa39/dlx-75559/node_modules/@expo/fingerprint/build/Fingerprint.js',
    '/private/var/folders/00/m6b9k2y13rvgz5dn7y7_dtx00000gn/T/xfs-d992fa39/dlx-75559/node_modules/@expo/fingerprint/build/index.js',
    '/private/var/folders/00/m6b9k2y13rvgz5dn7y7_dtx00000gn/T/xfs-d992fa39/dlx-75559/node_modules/@expo/fingerprint/cli/build/runLegacyCLIAsync.js',
    '/private/var/folders/00/m6b9k2y13rvgz5dn7y7_dtx00000gn/T/xfs-d992fa39/dlx-75559/node_modules/@expo/fingerprint/cli/build/cli.js',
    '/private/var/folders/00/m6b9k2y13rvgz5dn7y7_dtx00000gn/T/xfs-d992fa39/dlx-75559/node_modules/@expo/fingerprint/bin/cli.js'
  ]
}
```

### Steps to reproduce

Run `yarn dlx @expo/fingerprint@0.13.0 fingerprint:generate apps/mobile`

### Environment

```text
expo-env-info 1.3.3 environment info:
    System:
      OS: macOS 15.5
      Shell: 5.9 - /bin/zsh
    Binaries:
      Node: 22.14.0 - ~/.nvm/versions/node/v22.14.0/bin/node
      Yarn: 4.9.1 - ~/.nvm/versions/node/v22.14.0/bin/yarn
      npm: 10.9.2 - ~/.nvm/versions/node/v22.14.0/bin/npm
      Watchman: 2025.04.28.00 - /opt/homebrew/bin/watchman
    Managers:
      CocoaPods: 1.16.2 - /opt/homebrew/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 24.5, iOS 18.5, macOS 15.5, tvOS 18.5, visionOS 2.5, watchOS 11.5
    IDEs:
      Android Studio: 2024.3 AI-243.24978.46.2431.13363775
      Xcode: 16.4/16F6 - /usr/bin/xcodebuild
    npmPackages:
      expo: 53.0.10 => 53.0.10 
      expo-router: 5.0.7 => 5.0.7 
      expo-updates: 0.28.14 => 0.28.14 
      react: 19.1.0 => 19.1.0 
      react-native: 0.79.3 => 0.79.3 
    npmGlobalPackages:
      eas-cli: 16.7.0
    Expo Workflow: bare
```

### Expo Doctor Diagnostics

```text
12/15 checks passed. 3 checks failed. Possible issues detected:
Use the --verbose flag to see more details about passed checks.

✖ Check package.json for common issues
The following scripts in package.json conflict with the contents of node_modules/.bin: fingerprint.
Advice:
Update your package.json to remove conflicts.

✖ Validate packages against React Native Directory package metadata
The following issues were found when validating your dependencies against React Native Directory:
  No metadata available: ...
Advice:
Update React Native Directory to include metadata for unknown packages. Alternatively, set expo.doctor.reactNativeDirectoryCheck.listUnknownPackages in package.json to false to skip warnings about packages with no metadata, if the warning is not relevant.

✖ Check that packages match versions required by installed Expo SDK
The following packages should be updated for best compatibility with the installed expo version:
...
Your project may not work correctly until you install the expected versions of the packages.
Found outdated dependencies
Advice:
Use 'npx expo install --check' to review and upgrade your dependencies.

3 checks failed, indicating possible issues with the project.
```

--- a/packages/@expo/fingerprint/CHANGELOG.md
+++ b/packages/@expo/fingerprint/CHANGELOG.md
@@ -8,6 +8,8 @@
 
 ### 🐛 Bug fixes
 
+- Move `glob` from `devDependencies` to `dependencies` as its used outside test code.
+
 ### 💡 Others
 
 ## 0.13.0 - 2025-06-08

--- a/packages/@expo/fingerprint/CHANGELOG.md
+++ b/packages/@expo/fingerprint/CHANGELOG.md
@@ -8,7 +8,7 @@
 
 ### 🐛 Bug fixes
 
-- Move `glob` from `devDependencies` to `dependencies` as its used outside test code.
+- Move `glob` from `devDependencies` to `dependencies` as its used outside test code. ([#37332](https://github.com/expo/expo/pull/37332) by [@byCedric](https://github.com/byCedric))
 
 ### 💡 Others
 

--- a/packages/@expo/fingerprint/package.json
+++ b/packages/@expo/fingerprint/package.json
@@ -49,6 +49,7 @@
     "debug": "^4.3.4",
     "find-up": "^5.0.0",
     "getenv": "^2.0.0",
+    "glob": "^10.4.2",
     "ignore": "^5.3.1",
     "minimatch": "^9.0.0",
     "p-limit": "^3.1.0",
@@ -58,7 +59,6 @@
   "devDependencies": {
     "@types/find-up": "^4.0.0",
     "expo-module-scripts": "^4.1.7",
-    "glob": "^10.4.2",
     "require-from-string": "^2.0.2",
     "temp-dir": "^2.0.0"
   }

